### ISSUE TITLE
bug: Alt text missing for footer images

### ISSUE THREAD

**Issue by cathy-codes** [WomenCodingCommunity/WomenCodingCommunity.github.io#111]
**Task**
Add alt text for the SVGs inside the links to social media (LinkedIn, Slack, Github, Instagram, email) so that screen reader users can understand where they are going.

Here is an article on accessible SVGs: [https://www.smashingmagazine.com/2021/05/accessible-svg-patterns-comparison/#basic-alternative-descriptions-using-the-svg-tag](https://www.smashingmagazine.com/2021/05/accessible-svg-patterns-comparison/#basic-alternative-descriptions-using-the-svg-tag)

I propose to use Pattern 5:

```
<svg role="img" ...>
   <title>LinkedIn</title>
   [design code]
</svg>
```

**Comment by Rclarkeweb** [comment]
Hi, I would like to help with this if it’s still available? 😊

**Comment by lilyndk** [comment]
Thanks @Rclarkeweb, ticket is now assigned to you :)

**Comment by Rclarkeweb** [comment]
Thank you! Would it be possible to message someone on Slack about a few questions I have, or should I ask here? Just want to check a few things regarding this ticket.

**Comment by cathy-codes** [comment]
Hey @Rclarkeweb, would be good if you could ask here since it might help others learn something too but Slack is also fine.

**Comment by Rclarkeweb** [comment]
Thanks, I’ll ask here. As you want to use pattern 5 (as suggested in the first comment) would this look like the following for the icons: 
```
<svg xmlns="http://www.w3.org/2000/svg" class="icon" viewBox="0 0 16 16">
  <title>Instagram</title>
  <path d=“path is here"/>
</svg> 
```
Does it need to include `role="img"`?
Will I need to run the project locally as per the readme? Or can I make the changes without needing to do this? 
Sorry for all the questions - I’m still very new to open source contributions

**Comment by cathy-codes** [comment]
@Rclarkeweb Yep that looks good and `role="img"` would be important to include so screenreaders recognise it's an image of Instagram.

You'll have to run the project locally because we require that the changes be tested locally before merging into the project just to make sure things work.

No worries about asking questions! We're all here to help each other learn :)

**Comment by Rclarkeweb** [comment]
Hi, I have made the changes in my local repo but I'm having trouble installing Ruby to run and do the tests, will I need the Ruby version with Devkit or without? I've never used Ruby before so I'm a little unsure. I have a windows machine.

**Comment by cathy-codes** [comment]
@Rclarkeweb It was my first time with Ruby too (also on Windows), I used the RubyInstaller and since I wasn't sure about the version, the website recommended Ruby+Devkit 3.2.X (x64) - so with the Devkit.

**Comment by lilyndk** [comment]
Hi @Rclarkeweb, fyi a lot of component refactoring changes have been merged today related to branding re-design, thus please ensure to pull down latest changes from main branch, thanks.

**Comment by Rclarkeweb** [comment]
Thanks @lilyndk and @cathy-codes! I'm having trouble downloading Ruby, my computer won't let me do it because it says Ruby installer is unsafe? @cathy-codes did you have this problem?

**Comment by cathy-codes** [comment]
Hi @Rclarkeweb, are you experiencing something like this? If yes, then I pressed on More Info then Run Anyway.

![Screenshot 2024-05-13 151424](https://github.com/WomenCodingCommunity/WomenCodingCommunity.github.io/assets/11363288/32456371-8f05-4fff-924a-21aab073c776)
![Screenshot 2024-05-13 151436](https://github.com/WomenCodingCommunity/WomenCodingCommunity.github.io/assets/11363288/3d4861c0-62fd-4244-8600-c3d2081b6bc7)

**Comment by Rclarkeweb** [comment]
@cathy-codes yes, thank you!!

**Comment by Rclarkeweb** [comment]
Hi @lilyndk @cathy-codes, just to let you both know I have submitted a PR with the added alt text. Hope it's ok. 😄
